File: APPManage/AM.Model/basket_install.cs
/**  版本信息模板在安装目录下，可自行修改。
* basket_install.cs
*
* 功 能： N/A
* 类 名： basket_install
*
* Ver    变更日期             负责人  变更内容
* ───────────────────────────────────
* V0.01  2016/8/12 17:55:15   N/A    初版
*
* Copyright (c) 2012 Maticsoft Corporation. All rights reserved.
*┌──────────────────────────────────┐
*│　此技术信息为本公司机密信息，未经本公司书面同意禁止向第三方披露．　│
*│　版权所有：动软卓越（北京）科技有限公司　　　　　　　　　　　　　　│
*└──────────────────────────────────┘
*/
using System;
namespace AM.Model
{
	/// <summary>
	/// basket_install:实体类(属性说明自动提取数据库字段的描述信息)
	/// </summary>
	[Serializable]
	public partial class basket_install
	{
		public basket_install()
		{}
		#region Model
		private string _guid;
		private string _userid;
		private DateTime? _createdate;
		private string _gcmc;
		private string _sgdd;
		private string _sydw;
		private string _jldw;
		private string _sbmc;
		private string _azdw;
		private string _sbxh;
		private string _sbbh;
		private string _sccj;
		private string _aqsbh;
		private string _tzsbsczzxkz;
		private string _cqbh;
		private string _ccrq;
		private string _jygd;
		private string _szzobbh;
		private string _szzobyqzk;
		private string _wsdjbh;
		private string _wsdjyqzk;
		private string _szwybbh;
		private string _szwybyqzk;
		private string _scbh;
		private string _scyqzk;
		private string _gzcbh;
		private string _gzcyqzk;
		private string _szjdcbh;
		private string _szjdcyqzk;
		private string _fsybh;
		private string _fsyyqzk;
		private string _ybkcbh;
		private string _ybkcyqzk;
		private string _gjc30mbh;
		private string _gjc30mzxkz;
		private string _gjc5mbh;
		private string _gjc5mzxkz;
		private string _bzxmbhgs;
		private string _ybxmbhgs;
		/// <summary>
		/// 
		/// </summary>
		public string Guid
		{
			set{ _guid=value;}
			get{return _guid;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string userid
		{
			set{ _userid=value;}
			get{return _userid;}
		}
		/// <summary>
		/// 
		/// </summary>
		public DateTime? CreateDate
		{
			set{ _createdate=value;}
			get{return _createdate;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string gcmc
		{
			set{ _gcmc=value;}
			get{return _gcmc;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string sgdd
		{
			set{ _sgdd=value;}
			get{return _sgdd;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string sydw
		{
			set{ _sydw=value;}
			get{return _sydw;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string jldw
		{
			set{ _jldw=value;}
			get{return _jldw;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string sbmc
		{
			set{ _sbmc=value;}
			get{return _sbmc;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string azdw
		{
			set{ _azdw=value;}
			get{return _azdw;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string sbxh
		{
			set{ _sbxh=value;}
			get{return _sbxh;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string sbbh
		{
			set{ _sbbh=value;}
			get{return _sbbh;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string sccj
		{
			set{ _sccj=value;}
			get{return _sccj;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string aqsbh
		{
			set{ _aqsbh=value;}
			get{return _aqsbh;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string tzsbsczzxkz
		{
			set{ _tzsbsczzxkz=value;}
			get{return _tzsbsczzxkz;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string cqbh
		{
			set{ _cqbh=value;}
			get{return _cqbh;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string ccrq
		{
			set{ _ccrq=value;}
			get{return _ccrq;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string jygd
		{
			set{ _jygd=value;}
			get{return _jygd;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string szzobbh
		{
			set{ _szzobbh=value;}
			get{return _szzobbh;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string szzobyqzk
		{
			set{ _szzobyqzk=value;}
			get{return _szzobyqzk;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string wsdjbh
		{
			set{ _wsdjbh=value;}
			get{return _wsdjbh;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string wsdjyqzk
		{
			set{ _wsdjyqzk=value;}
			get{return _wsdjyqzk;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string szwybbh
		{
			set{ _szwybbh=value;}
			get{return _szwybbh;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string szwybyqzk
		{
			set{ _szwybyqzk=value;}
			get{return _szwybyqzk;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string scbh
		{
			set{ _scbh=value;}
			get{return _scbh;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string scyqzk
		{
			set{ _scyqzk=value;}
			get{return _scyqzk;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string gzcbh
		{
			set{ _gzcbh=value;}
			get{return _gzcbh;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string gzcyqzk
		{
			set{ _gzcyqzk=value;}
			get{return _gzcyqzk;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string szjdcbh
		{
			set{ _szjdcbh=value;}
			get{return _szjdcbh;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string szjdcyqzk
		{
			set{ _szjdcyqzk=value;}
			get{return _szjdcyqzk;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string fsybh
		{
			set{ _fsybh=value;}
			get{return _fsybh;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string fsyyqzk
		{
			set{ _fsyyqzk=value;}
			get{return _fsyyqzk;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string ybkcbh
		{
			set{ _ybkcbh=value;}
			get{return _ybkcbh;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string ybkcyqzk
		{
			set{ _ybkcyqzk=value;}
			get{return _ybkcyqzk;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string gjc30mbh
		{
			set{ _gjc30mbh=value;}
			get{return _gjc30mbh;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string gjc30mzxkz
		{
			set{ _gjc30mzxkz=value;}
			get{return _gjc30mzxkz;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string gjc5mbh
		{
			set{ _gjc5mbh=value;}
			get{return _gjc5mbh;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string gjc5mzxkz
		{
			set{ _gjc5mzxkz=value;}
			get{return _gjc5mzxkz;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string bzxmbhgs
		{
			set{ _bzxmbhgs=value;}
			get{return _bzxmbhgs;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string ybxmbhgs
		{
			set{ _ybxmbhgs=value;}
			get{return _ybxmbhgs;}
		}
		#endregion Model

	}
}

File: AM/Scripts/appm/basket_install.js
function test() {
    var isok = false;
    if ($("#gcname").val() == "") {
        alert("工程名称不能为空");
        $("#gcname").focus();
    } else {
        if ($("#sgaddress").val() == "") {
            alert("施工地点不能为空");
            $("#sgaddress").focus();
        } else {
            if ($("#sycompany").val() == "") {
                alert("使用单位不能为空");
                $("#sycompany").focus();
            } else {
                if ($("#jlcompany").val() == "") {
                    alert("监理单位不能为空");
                    $("#jlcompany").focus();
                } else {
                    if ($("#sbname").val() == "") {
                        alert("设备名称不能为空");
                        $("#sbname").focus();
                    } else {
                        if ($("#azcompany").val() == "") {
                            alert("安装单位不能为空");
                            $("#azcompany").focus();
                        } else {
                            if ($("#type").val() == "") {
                                alert("设备型号不能为空");
                                $("#type").focus();
                            } else {
                                if ($("#sbcode").val() == "") {
                                    alert("设备编号不能为空");
                                    $("#sbcode").focus();
                                } else {
                                    if ($("#scstore").val() == "") {
                                        alert("生产厂家不能为空");
                                        $("#scstore").focus();
                                    } else {
                                        if ($("#aqscode").val() == "") {
                                            alert("安全锁编号不能为空");
                                            $("#aqscode").focus();
                                        } else {
                                            if ($("#xkz").val() == "") {
                                                alert("特种设备生产制造许可证不能为空");
                                                $("#xkz").focus();
                                            } else {
                                                if ($("#cqcode").val() == "") {
                                                    alert("产权编号不能为空");
                                                    $("#cqcode").focus();
                                                } else {
                                                    if ($("#ccdate").val() == "") {
                                                        alert("出厂日期不能为空");
                                                        $("#ccdate").focus();
                                                    } else {
                                                        if ($("#checkheight").val() == "") {
                                                            alert("检验高度不能为空");
                                                            $("#checkheight").focus();
                                                        } else
                                                            isok = true;
                                                    }
                                                }
                                            }
                                        }
                                    }
                                }
                            }
                        }
                    }
                }
            }
        }
    }
    return isok;
}

String.prototype.trim = function () {

    return this.replace(/(^\s*)|(\s*$)/g, '');
}

function submit() {
    //var check = test();
    //if (check)
    //{
    var userid = $("#userid").val();
    var objArr = $("input[type=text]")
    
    //var checkitem = $("input[name=checkbox_name]:checked");
    //var radios = $("input[name=sex]:checked");
    //var xb = $(radios[0]).next("label").text();
    //var textareas = $("textarea");
    //var pxd = "";
    //for (var i = 0; i < checkitem.length; i++) {
        
    //    pxd += $(checkitem[i]).parent("label").text().trim();
    //    if (i < checkitem.length) {
    //        pxd += ","
    //    }
    //}
    var json = {
        "basket_install": [{
            "Guid": "",
            "CreateDate": "",
            "userid": userid,
            "gcmc":$(objArr[0]).val(),
            "sgdd":$(objArr[1]).val(),
            "sydw":$(objArr[2]).val(),
            "jldw":$(objArr[3]).val(),
            "sbmc":$(objArr[4]).val(),
            "azdw":$(objArr[5]).val(),
            "sbxh":$(objArr[6]).val(),
            "sbbh":$(objArr[7]).val(),
            "sccj":$(objArr[8]).val(),
            "aqsbh":$(objArr[9]).val(),
            "tzsbsczzxkz":$(objArr[10]).val(),
            "cqbh":$(objArr[11]).val(),
            "ccrq":$(objArr[12]).val(),
            "jygd":$(objArr[13]).val(),
            "szzobbh":$(objArr[14]).val(),
            "szzobyqzk":$(objArr[15]).val(),
            "wsdjbh":$(objArr[16]).val(),
            "wsdjyqzk":$(objArr[17]).val(),
            "szwybbh":$(objArr[18]).val(),
            "szwybyqzk":$(objArr[19]).val(),
            "scbh":$(objArr[20]).val(),
            "scyqzk":$(objArr[21]).val(),
            "gzcbh":$(objArr[22]).val(),
            "gzcyqzk":$(objArr[23]).val(),
            "szjdcbh":$(objArr[24]).val(),
            "szjdcyqzk":$(objArr[25]).val(),
            "fsybh":$(objArr[26]).val(),
            "fsyyqzk":$(objArr[27]).val(),
            "ybkcbh":$(objArr[28]).val(),
            "ybkcyqzk":$(objArr[29]).val(),
            "gjc30mbh":$(objArr[30]).val(),
            "gjc30mzxkz":$(objArr[31]).val(),
            "gjc5mbh":$(objArr[32]).val(),
            "gjc5mzxkz":$(objArr[33]).val(),
            "bzxmbhgs":$(objArr[34]).val(),
            "ybxmbhgs": $(objArr[35]).val(),
        }]
    };
    $.ajax({
        type: "post",
        url: "/AppM/addbasket_install",
        data: { obj: JSON.stringify(json) },
        success: function (data) {
            layer.msg("提交成功");

        }
    })
    
    //}
}